src/Umbraco.Core/Models/ISimpleContentType.cs

```diff
@@ -15,6 +15,8 @@ public interface ISimpleContentType : IUmbracoEntity
         /// </summary>
         string Alias { get; }
 
+        string Description { get; }
```

Comment:
Hrm, we'll have to see what implications this has since it's a breaking change. In theory we are allowed to change this in accordance with our breaking change policies so just need to see if people implement this... i don't think so, but could be wrong.